src/main/java/com/three_tech_solutions/slot_app/controllers/interfaces/StudentController.java

```diff
@@ -0,0 +1,15 @@
+package com.three_tech_solutions.slot_app.controllers.interfaces;
+
+import com.three_tech_solutions.slot_app.dto.StudentRequestDTO;
+import com.three_tech_solutions.slot_app.dto.StudentResponseDTO;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+//definimos el path y el metodo HTTP en la interfaz.
+@RequestMapping("/students")
+public interface StudentController {
+    @PostMapping
+    ResponseEntity<StudentResponseDTO> createStudent(@Valid @RequestBody StudentRequestDTO studentDTO);
```

Comment:
ResponseEntity es una clase que nos permite agregar distinta informacion a nuestra respuesta HTTP, como por ejemplo, el HTTP Status, el payload (body) o los headers. Si bien no esta mal usar ResponseEntity, no vale la pena usarlo simplemente para definir el tipo de HTTP Status que queremos retornar si todo sale bien, para eso existe un decorador llamado `@ResponseStatus`, de modo tal que podriamos modificar esto de la siguiente manera

```suggestion
    @PostMapping
    @ResponseStatus(HttpStatus.CREATED)
    StudentResponseDTO createStudent(@Valid @RequestBody StudentRequestDTO studentDTO);
```

Fijate que el tipo de retorno ahora es un `StudentResponseDTO` y ya no es mas un `ResponseEntity<StudentResponseDTO>`